configs/data_gatherer_localization.conf

```diff
@@ -0,0 +1,28 @@
+# Path to where to store the recorded data.
+--data_path=data/
+# Collect imu data.
+--log_imu
+# Collect gnss data.
+--log_gnss
+# Collect pose data.
+--log_pose
+# Set skip cameras.
+--skip_cameras
+###### Control config #####
+--control=simulator_auto_pilot
+############# Carla settings #############
+--simulator_timeout=60
+--simulator_num_people=225
+--simulator_num_vehicles=50
+# Start at a random spawn point.
+--simulator_spawn_point_index=-1
+--stop_for_traffic_lights=False
+--camera_image_width=1920
+--camera_image_height=1080
```

Comment:
These flags don't realy matter given that you're not adding any cameras, and that you're using the simulator auto pilot. Stop for traffic light only affects the behavior of the ego vehicle when you're using the Pylot control to drive around.